static/src/javascripts/projects/common/modules/commercial/geo-utils.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import { getSync as geolocationGetSync } from 'lib/geolocation';
+
+let geo;
+const currentGeoLocation = ((): string => {
+    geo = geo || geolocationGetSync();
```

Comment:
Note we have lodash 'once' for this, but check 'geolocationGetSync()' is not 'onced' already :)

Reply:
Yeap, this is our version of `once`, as wanted to avoid relying on lodash. It should do the same thing